pkgs/development/compilers/gcc/5/default.nix

```diff
@@ -547,4 +547,10 @@ stdenv.mkDerivation ({
 // optionalAttrs (!stripped || cross != null) { dontStrip = true; NIX_STRIP_DEBUG = 0; }
 
 // optionalAttrs (enableMultilib) { dontMoveLib64 = true; }
+
+# This is only here because it was here in previous commits and we
+# want to avoid a meaningless mass rebuild.  These variables get
+# overridden via crossAttrs in an actual cross-build.
+// { crossMingw = false; crossStageStatic = true; }
+
```

Comment:
I thought that adding this would make it simpler and easier for my pull request to be merged, and it also helped me ensure that I did not accidentally screw something up.  I used Nix assertions to [assert](https://gist.github.com/DavidEGrayson/8d61032e523b3bc8bbca0c4b281afb52) that the native GCC in my pull request and in the upstream commit it is based on are the same :-).

This is kind of a subjective decision, so I would mainly be concerned with what the people in charge of nixpkgs think.  @groxxda, do you authority to merge my pull request?